packages/create-llama/index.ts

```diff
@@ -111,20 +111,27 @@ const program = new Commander.Command(packageJson.name)
     `
 
   Select OpenAI model to use. E.g. gpt-3.5-turbo.
+`,
```

Comment:
It's a good idea to make both ports configurable, but now it seems we're having a naming problem.
As the user of `create-llama`, there are two ports:

1. the `appPort`
2. the `backendPort` (optional)

if there's no frontend, we just have the app port, if there is a frontend, the frontend will use the `appPort` and connect to the backend via the `backendPort`.

Can you please check that the naming is consistent inside of `create-llama`?

Reply:
Yes, it is consistent, but we already used `externalPort` and `port` so I did not change the name. In some functions, i mapped them internally to backendPort and frontendPort for better code logic.